### PR TITLE
Install phantomjs in the browser role TE-884

### PR DESCRIPTION
**Pull request by jzoldak**
@benpatterson

### CODE REVIEW

**Comment by benpatterson** [comment]
LGTM.

File: playbooks/roles/browsers/tasks/main.yml [hunk 1]
```diff
@@ -15,20 +15,54 @@
...
+
+- name: move PhantomJS binary to /usr/local
+  shell: mv /var/tmp/{{ phantomjs_tarfile }}/bin/phantomjs /usr/local/bin/phantomjs
+  when: download_phantom_js.changed
+
+- name: verify phantomjs location
+  stat: path=/usr/local/bin/phantomjs
+  register: phantomjs
+- assert:
+    that: "phantomjs.stat.exists"
```
Comment (benpatterson) [inline playbooks/roles/browsers/tasks/main.yml:65]: Needs to be in quotes for ansible, eh? That's kinda weird.

**Comment by benpatterson** [comment]
:+1:

File: playbooks/roles/browsers/tasks/main.yml (hunk 1 above)
Comment (jzoldak) [inline playbooks/roles/browsers/tasks/main.yml:65]: Actually looking at the examples again, it seems the quotes are optional if you don't have spaces or single quotes in the statement. Reference: [Testing Strategies in the Ansible docs](http://docs.ansible.com/test_strategies.html).

If we're establishing a style preference, what's your opinion - should I remove them? I don't have a strong opinion either way.

Comment (benpatterson) [inline playbooks/roles/browsers/tasks/main.yml:65]: I don't have a strong preference, either. Sounds like "sometimes you need quotes". If that's the case, and we are landing on a style, might as well always use quotes.

Comment (jzoldak) [inline playbooks/roles/browsers/tasks/main.yml:65]: "lolz." Alright I'll leave them. :)

**Comment by jzoldak** [comment]
@benpatterson Jenkins worker ami build job [passed successfully](https://build.testeng.edx.org/job/build-jenkins-worker-ami/30/). Merging now.

**Comment by benpatterson** [comment]
w00t

**Pull request merged**